ui/pom.xml

```diff
@@ -104,5 +106,195 @@
+        </profile>
+        <profile>
+            <id>with-custom-cafile</id>
+            <activation>
+                <property>
+                <name>custom-cafile</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <executions>
+                    <execution>
+                        <id>yarn set cafile</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                        <goal>yarn</goal>
+                        </goals>
+                        <configuration>
+                        <arguments>config set cafile ${custom-cafile}</arguments>
+                        </configuration>
+                    </execution>
+                    </executions>
+                </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>insecure</id>
+            <build>
+            <plugins>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
+                    <artifactId>frontend-maven-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>yarn set no-strict ssl</id>
+                            <phase>generate-sources</phase>
+                            <goals>
+                            <goal>yarn</goal>
+                            </goals>
+                            <configuration>
+                            <arguments>config set strict-ssl false</arguments>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>verbose</id>
+            <properties>
+                <yarn-verbose>--verbose</yarn-verbose>
+                <npm-verbose>--loglevel=silly</npm-verbose>
+            </properties>
+        </profile>
+        <profile>
+            <id>macosx</id>
+            <activation>
+                <os><family>mac</family></os>
+            </activation>
+            <properties>
+                <os.type>darwin</os.type>
+            </properties>
+        </profile>
+        <profile>
+            <id>linux</id>
+            <activation>
+                <os><name>Linux</name></os>
+            </activation>
+            <properties>
+                <os.type>linux</os.type>
+            </properties>
+        </profile>
+        <profile>
+            <id>windows</id>
+            <activation>
+                <os><name>Windows</name></os>
+            </activation>
+            <properties>
+                <os.type>win32</os.type>
+            </properties>
+        </profile>
```

Comment:
What are these for?  I see they are setting the `os.type` property but I can't see where that property is used.  Used implicitly somewhere?

Reply:
You're right.    One of the proxy profiles that is used in syndesis utilized these, but it's not necessary for a-r.     I'll remove them, it doesn't make sense to include them.